2016/ideas-list-swc.md

```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
 Thanks to @xuf12 for the initial idea behind this project and for @wking for
 providing this idea for GSOC'15.
 
+---------
+
+## Manage workflow for instructor training
+
+**Please ask questions as issues [here][gsoc-issues].**
+
+### Abstract
+
+Extend [AMY][] to manage the process of training people to be instructors.
+
+| **Intensity** | **Involves**  | **Mentors** |
+| ------------- | --------------|------------ |
+| Moderate | Workflow Design, Interface Design, Python, Django, Testing | [@pbanaszkiewicz][] |
+
+### Background
+
+[Software Carpentry][swc] has built a Django-based web application called [AMY][]
+to keep track of who's qualified to teach,
+what workshops have run,
+who taught and attended,
+and so on.
+This project will extend it to manage all aspects of instructor training,
+and will replace the current mish-mash of Etherpads and Google Docs spreadsheets
+that we use for this purpose.
+The main requirements are:
+
+* recording applications from individuals and groups who wish to become trainers
+* matching those applications to upcoming instructor training workshops
+* keeping track of which stages of instructor training have been completed by specific individuals
+
+### Technical details
+
+Challenges for this project: analyzing the current workflow
+(which relies on a lot of email, some Etherpads, and a Google Docs spreadsheet)
+to determine what we're actually doing;
+translating that workflow into new pages in [AMY][]
+that can be used by administrators, trainers, and mentors;
+making that workflow's implementation flexible so that it can easily be reconfigured as our process evolves;
+testing both the usability and functionality of the new interface.
+
+The software stack for this project is [Python][], [Django][], and [JQuery][].
+
+## Open Source Development Experience
```

Comment:
I don't know how that header works. Is the proposal supposed to be divided in two parts, @rgaiacs? I know this pattern repeats in other proposals too.

I'd change this header to `###`.